tests/e2e/esm/index.test.js

```diff
@@ -278,3 +287,38 @@ describe('ANTRegistry', async () => {
     assert(Array.isArray(affiliatedAnts.Controlled));
   });
 });
+
+describe('Signing', async () => {
+  let signers = [];
+  before(async () => {
+    const jwk = await arweave.wallets.generate();
```

Comment:
nit: this is an expensive operation and creates side effects - can we just load a test wallet into the repo and use the same one every time?